spec/integration/outputs/create_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe "client create actions", :integration => true do
   require "logstash/outputs/elasticsearch"
 
-  def get_es_output(action, id)
+  def get_es_output(action, id, external_version)
```

Comment:
Can we just default the new parameter to `nil` or `false` to keep the 2-arity version valid?